src/spec/payid-easy-checkout.md

```diff
@@ -0,0 +1,378 @@
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC2119][] and [RFC9174][].
+
+# Introduction
+
+The PayID Easy Checkout Protocol is a minimal protocol that allows a recipient (e.g., an online merchant or a charity) to
+request a payment from a payer using only the payer's PayID as defined in [PAYID-URI][]. Implementations
+of this protocol require little to no server-side engineering efforts, while creating a seamless and uniform
+user experience for payers.
+
+The main focus of the protocol is on PayID Easy Checkout Discovery, which defines how a PayID Easy Checkout Client can 
+use a PayID to retrieve a PayID Easy Checkout URL which represents a resource that the payer's digital wallet can use to initiate 
+a payment to the merchant. 
+
+Though the [appendix](#appendix) of this specification provides an example usage of a 
+PayID Easy Checkout URL using Web Redirects, supplemental RFCs are needed to define the different ways in which a PayID
+client can utilize a PayID Easy Checkout URL.
+
+# PayID Easy Checkout Protocol
+The PayID Easy Checkout Protocol can be used to facilitate an end-to-end checkout flow between a payment recipient, such
+as an online merchant, and a payer via a sending client, such as a wallet.
+
+The protocol is comprised of two parts:
+
+1. PayID Easy Checkout Discovery
+2. PayID Easy Checkout URL Assembly
+
+## PayID Easy Checkout Discovery
+PayID Easy Checkout Discovery extends [PAYID-DISCOVERY][] by defining a new link in the PayID metadata JRD
+returned by a PayID Discovery query. This link, defined in the [JRD section](#payid-easy-checkout-jrds) 
+of this specification, includes the PayID Easy Checkout URL representing a resource on the wallet which can
+be used to complete a payment.
+
+Recipients who wish to initiate an Easy Checkout flow MUST query the sender's PayID Discovery Server to 
+obtain a PayID Easy Checkout URL. PayID Discovery Servers that wish to enable PayID Easy
+Checkout MUST include a JRD Link conforming to the definition in the [JRD section](#payid-easy-checkout-jrds) of this document
+in all PayID Easy Checkout Discovery responses.
+
+Recipients SHOULD implement fallback measures to complete a checkout flow if a payer's wallet does not support PayID Easy Checkout.
+
+The following steps describe how a PayID Easy Checkout Client can query a PayID Discovery Server to obtain a PayID Easy Checkout URL.
+
+### Step 1: Assemble PayID Easy Checkout Discovery URL
+The process of assembling a PayID Discovery URL is defined in section 4.1.1 of [PAYID-DISCOVERY][], and is
+the same as for this protocol.
+
+### Step 2: Query PayID Easy Checkout Discovery URL
+Querying the PayID Discovery URL is defined in section 4.1.2 of [PAYID-DISCOVERY][], and is performed
```

Comment:
The process of querying the PayID Discovery URL is defined in section 4.1.2 of [PAYID-DISCOVERY][].